project/Dependencies.scala

```diff
@@ -3,7 +3,7 @@ package build
 object Dependencies {
   val Scala210Version = "2.10.7"
   val Scala211Version = "2.11.12"
-  val Scala212Version = "2.12.8"
+  val Scala212Version = "2.12.13"
```

Comment:
Do we need this change? I haven't managed to bump the version due to some issues with native image, which is causing the issues in CI

Reply:
Actually no I don't need this change since I can choose a different Scala version in the tests.